agent/conf/agent.properties

```diff
@@ -263,3 +263,7 @@ iscsi.session.cleanup.enabled=false
 # Automatically clean up iscsi sessions not attached to any VM.
 # Should be enabled for users using managed storage for example solidfire.
 # Should be disabled for users with unmanaged iscsi connections on their hosts
+
+# This parameter specifies the heartbeat update timeout in ms.
+# Depending on the use case, this timeout might need increasing/decreasing.
+# heartbeat.update.timeout=60000
```

Comment:
I suppose 60000 is the default (1 min) wouldn't hurt to make that explicit in here for operator convenience.

Reply:
@DaanHoogland done.